### ISSUE TITLE
bokehjs tests are failing on regular basis

### ISSUE THREAD

**Issue by mattpap** [bokeh/bokeh#5063]
```
SidePanel.Model update_constraints "before each" hook for "should not fail if visible is not on model":
Error: timeout of 2000ms exceeded. Ensure the done() callback is being called in this test.
```

**Comment by birdsarah** [comment]
I believe I made these tests. Short-term fix is to skip them.

But this and others like it, that are already skipped, need fixing so they behave themselves.

**Comment by birdsarah** [comment]
The error on the other tests seems to some sort of (maybe race) condition where the stubbed solver doesn't get used in time.

I'm not sure about this one.

**Comment by mattpap** [comment]
This issue was fixed in PR #5838.
